var/spack/repos/builtin/packages/cryptsetup/package.py

```diff
@@ -25,7 +25,7 @@ class Cryptsetup(AutotoolsPackage):
     depends_on('lvm2', type=('build', 'link'))
     depends_on('popt', type=('build', 'link'))
     depends_on('json-c', type=('build', 'link'))
-    depends_on('util-linux~libuuid', type=('build', 'link'))
+    depends_on('util-linux', type=('build', 'link'))
```

Comment:
This was changed in #12839 to fix an issue with libuuid. Why do you want to revert it?